cmd/nodecmd/create.go

```diff
@@ -97,6 +98,7 @@ will apply to all nodes in the cluster`,
 	cmd.Flags().StringVar(&awsProfile, "aws-profile", constants.AWSDefaultCredential, "aws profile to use")
 	cmd.Flags().BoolVar(&createOnFuji, "fuji", false, "create node/s in Fuji Network")
 	cmd.Flags().BoolVar(&createDevnet, "devnet", false, "create node/s into a new Devnet")
+	cmd.Flags().IntVar(&devnetNumRPCNodes, "devnet-api-nodes", 1, "number of API nodes(nodes without stake) to create in the new Devnet")
```

Comment:
pls make this 0 as default, having API nodes should not be the default devnet setting, it is specific request

Reply:
I think POG actually wants to have API nodes as default for devnet, but I am ok with changing default to 0